### PR TITLE
[SOL-2045] Create proper error pages when catalog service not available.

### PR DESCRIPTION
**Pull request by iivic**
screen shot 2016-09-15 at 10 07 44
@marjev  @vkaracic  @mjfrey

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @iivic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=940

**Review by vkaracic — APPROVED** [review]
LGTM

**Comment by marjev** [comment]
LGTM 👍

File: ecommerce/coupons/views.py [hunk 1]
```diff
@@ -127,6 +129,10 @@ def get_context_data(self, **kwargs):
                 return {
                     'error': _('The voucher is not applicable to your current basket.'),
                 }
+            except (ConnectionError, SlumberBaseException, Timeout):
+                return {
+                    'error': _('Course discovery service is not available at the moment.'),
```
Comment (mjfrey) [inline ecommerce/coupons/views.py]: "Coupon information not available at this time.  Please try again later."

**Comment by mjfrey** [comment]
One small text change.
👍

**Pull request merged**